source/_integrations/timer.markdown

```diff
@@ -207,3 +266,47 @@ script:
         target:
           entity_id: timer.test
 ```
+
+### Motion light with auto-off
+
+The light turns on when motion is detected and stays on while movement continues.
+When motion stops and the timer finishes, the light turns off.
+
+```yaml
+alias: Motion-Activated Hall Light
```

Comment:
Here I specifically used trigger_id to have an example of how to handle two events in one automation, which is useful when you manages automations in the UI.